<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Launcher</class>
 <widget class="QDialog" name="Launcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Aircraft</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Search:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QListView" name="aircraftList"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="aircraftFilter">
         <property name="placeholderText">
          <string>Search aircraft</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="customAircraftDirLabel">
         <property name="text">
          <string>Custom aircraft directory:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="openAircraftDirButton">
         <property name="text">
          <string>Open in Finder</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Location</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="7" column="3" colspan="3">
        <widget class="QComboBox" name="parkingCombo"/>
       </item>
       <item row="4" column="3" colspan="3">
        <widget class="QComboBox" name="runwayCombo"/>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QRadioButton" name="runwayRadio">
         <property name="text">
          <string>Runway:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="airportIdent">
         <property name="text">
          <string>Search:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QRadioButton" name="parkingRadio">
         <property name="text">
          <string>Parking:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="airportHistory">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>H</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="4">
        <widget class="QLabel" name="airportDescription">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="3" colspan="3">
        <widget class="QCheckBox" name="onFinalCheckbox">
         <property name="text">
          <string>On 10-mile final</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="airportEdit">
         <property name="placeholderText">
          <string>Enter an ICAO code or search by name</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="6">
        <widget class="AirportDiagram" name="airportDiagram" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="msaaCheckbox">
         <property name="text">
          <string>Enable Multi-sample anti-aliasing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="rembrandtCheckbox">
         <property name="text">
          <string>Enable deferred rendering (Rembrandt)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="terrasyncCheck">
         <property name="text">
          <string>Enable automatic scenery downloading (TerraSync)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fetchRealWxrCheckbox">
         <property name="text">
          <string>Fetch real weather online</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fullScreenCheckbox">
         <property name="text">
          <string>Start full-screen</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>412</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AirportDiagram</class>
   <extends>QWidget</extends>
   <header location="global">GUI/AirportDiagram.hxx</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
